tests/integration/ceph_bucket_notification_test.go

```diff
@@ -122,38 +110,189 @@ func (s *ObjectSuite) TestBucketNotificationsInOrder() {
 	})
 
 	t.Run("check CephBucketNotification created for bucket", func(t *testing.T) {
-		var s3client *rgw.S3Agent
-		s3endpoint, _ := helper.ObjectClient.GetEndPointUrl(namespace, storeName)
-		s3AccessKey, _ := helper.BucketClient.GetAccessKey(obcName)
-		s3SecretKey, _ := helper.BucketClient.GetSecretKey(obcName)
-		if objectStore.Spec.IsTLSEnabled() {
-			s3client, err = rgw.NewInsecureS3Agent(s3AccessKey, s3SecretKey, s3endpoint, rgwcontext.ZoneGroup, true)
-		} else {
-			s3client, err = rgw.NewS3Agent(s3AccessKey, s3SecretKey, s3endpoint, rgwcontext.ZoneGroup, true, nil)
-		}
+		notificationPresent := utils.Retry(12, 2*time.Second, "notification is created for bucket", func() bool {
+			return helper.BucketClient.CheckBucketNotificationSetonRGW(namespace, storeName, obcName, bucketname, notificationName, s3endpoint)
+		})
+		assert.True(t, notificationPresent)
+		logger.Info("CephBucketNotification created successfully on bucket")
+	})
+
+	t.Run("add non-notification label to OBC", func(t *testing.T) {
+		obc, err := k8sh.BucketClientset.ObjectbucketV1alpha1().ObjectBucketClaims(obcNamespace).Get(ctx, obcName, metav1.GetOptions{})
+		assert.Nil(t, err)
+		obc.Labels["test-label"] = "test-value"
+		_, err = k8sh.BucketClientset.ObjectbucketV1alpha1().ObjectBucketClaims(obcNamespace).Update(ctx, obc, metav1.UpdateOptions{})
+		assert.Nil(t, err)
+		// check whether existing bucket notification uneffected
+		notificationPresent := utils.Retry(12, 2*time.Second, "notification is created for bucket", func() bool {
+			return helper.BucketClient.CheckBucketNotificationSetonRGW(namespace, storeName, obcName, bucketname, notificationName, s3endpoint)
+		})
+		assert.True(t, notificationPresent)
+	})
+
+	t.Run("remove non-notification label from OBC", func(t *testing.T) {
+		obc, err := k8sh.BucketClientset.ObjectbucketV1alpha1().ObjectBucketClaims(obcNamespace).Get(ctx, obcName, metav1.GetOptions{})
+		assert.Nil(t, err)
+		delete(obc.Labels, "test-label")
+		_, err = k8sh.BucketClientset.ObjectbucketV1alpha1().ObjectBucketClaims(obcNamespace).Update(ctx, obc, metav1.UpdateOptions{})
+		assert.Nil(t, err)
+		// check whether existing bucket notification uneffected
+		notificationPresent := utils.Retry(12, 2*time.Second, "notification is created for bucket", func() bool {
+			return helper.BucketClient.CheckBucketNotificationSetonRGW(namespace, storeName, obcName, bucketname, notificationName, s3endpoint)
+		})
```

Comment:
Since removing a label isn't currently supported and this test is currently invalid, replace with TODO and `t.Skip()`.